shared/modules/mv3.utils.js

```diff
@@ -10,7 +13,7 @@ const runtimeManifest =
  * If this function is running in Node doing a build job, it will read process.env.ENABLE_MV3.
  * If this function is running in Node doing an E2E test, it will `fs.readFileSync` the manifest.json file.
  */
-const isManifestV3 = runtimeManifest
+export const isManifestV3 = runtimeManifest
```

Comment:
Converted to using the export syntax - will see if CI fails.

I'm looking at the commits for this file, and we constantly switch back and forth between module.exports and exports.

<details><summary>See Recording showcasing this files timeline</summary>


https://github.com/user-attachments/assets/416d517b-3502-4067-abf3-8ef70c9cc74d


</details>

Reply:
NVM lavamoat does not like this.